notebooks/jupyter_tutorial.md

```diff
@@ -0,0 +1,37 @@
+# Tutorial: Running a Jupyter Lab Workload
+A workload is the actual job or task you want to run on the platform. This could be training an AI model, running inference, doing data preprocessing, or conducting a scientific simulation.
```

Comment:
Also making the endpoint for models available via the web maybe?
Something like:
```suggestion
A workload is the actual job or task you want to run on the platform. This could be training an AI model, and running an inference model and exposing its endpoint, doing data preprocessing, or conducting a scientific simulation.
```